Source/ACE/Entity/Player.cs

```diff
@@ -467,12 +469,15 @@ public AceObject GetSavableCharacter()
             // Clone Character
             AceObject obj = (AceObject)Character.Clone();
 
+            // TODO: Og II this is where I need to load aceobject.inventory with the container.inventory
+
             // immediately after cloning, clear all dirty flags and HasBeenSavedToDatabase
             Character.ClearDirtyFlags();
 
             // These don't usually get saved back to the object so setting here for now.
             // Realisticly speaking, I think it will be possible to eliminate WeenieHeaderFlags and PhysicsDescriptionFlag from the datbase
             // AceObjectDescriptionFlag possibly could be eliminated as well... -Ripley
+            // actually we do use those without creating a wo - so it would be needed to keep them in the database Og II
```

Comment:
We have to store the flags because we are not always working from the world object.   Sometimes, we are just at AceObject when we need to know.   if we don't keep that, then introduce a crap load of complexity where you have to instantiate a world object just to see what it's flag is.   That is very limiting IMO.